### ISSUE TITLE
'utf8mb4_unicode_ci' is not valid for CHARACTER SET 'utf8'

### ISSUE THREAD

**Issue by fritzmg** [contao/contao#4798]
### Affected version(s)

4.13, 5.x

### Description

In #4592 we added `collation: utf8mb4_unicode_ci` to the config skeleton. However, many users have something like this in their `config.yaml`

```yaml
doctrine:
    dbal:
        connections:
            default:
                default_table_options:
                    charset: utf8
                    collate: utf8_unicode_ci
```

due to https://docs.contao.org/manual/en/installation/system-requirements/#mysql-minimum-requirements

The addition in the config skeleton in the manager-bundle will then create a mismatch between the `charset` and the `collation`, as the `charset` is set to `utf8`, but the `collation` is set to `utf8mb4_unicode_ci` - which will result in an

```
'utf8mb4_unicode_ci' is not valid for CHARACTER SET 'utf8'
```

error when updating the database in the Install Tool or via `contao:migrate`.

Can anyone think of a solution for this, other than updating the documentation and changing `collate` to `collation`? (I am not sure if _both_ are actually needed as the original related Doctrine issues are hard to follow.)

**Comment by kamalkech** [comment]
@leofeyer why close issue without post a comment ?? that s not helpful

**Comment by leofeyer** [comment]
<img width="727" alt="" src="https://github.com/contao/contao/assets/1192057/c3542991-4436-48e2-b9f1-24fa76a9d532">
